app/models/lunch.rb

```diff
@@ -60,9 +68,26 @@ def chalmrest
           summary, price = entry.summary.split('@')
           summary = summary.strip
           price = price.strip
-
+          if restaurant[:name] == "Express" || restaurant[:name] == "Kårrestaurangen"
+            allergens = ""
+            if entry[:summary].include? EGG_IMG_FILE_NAME
```

Comment:
- Iterate over allergens
- Move logic into method
- Use translated name instead of hard-coded strings